pkgs/applications/science/electronics/kicad/base.nix

```diff
@@ -165,6 +173,14 @@ stdenv.mkDerivation rec {
   ++ optional (debug) valgrind
   ;
 
+  postInstall = optionalString stdenv.isDarwin ''
```

Comment:
i don't fully understand what's being done here
but generally the approach is to do wrapping of the base package in the `default.nix` package that's exposed as `kicad`

Reply:
On macOS, KiCad binaries are bundled to `$out/*.app/Contents/MacOS/*` (e.g. `$out/KiCad.app/Contents/MacOS/kicad`). The `postInstall` in `kicad.base`

1. moves `$out/*.app` to `$out/Applications/*.app`, so that `nix-darwin` users would have `KiCad` in their Launchpad;
2. wraps `$out/Applications/*.app/Contents/MacOS/*` to `$out/bin/*`, allowing terminal access and making it compatible with `kicad`.

Reply:
doesn't this result in a doubly wrapped binary?
(i assume `kicad` would then wrap the apps in `$out/bin/` again)
i don't know if there's any problem with that, but it feels wrong to me